tests/pagination/tests.py

```diff
@@ -274,6 +274,19 @@ def test_get_page_empty_object_list_and_allow_empty_first_page_false(self):
         with self.assertRaises(EmptyPage):
             paginator.get_page(1)
 
+    def test_the_paginator_behaves_like_an_iterator(self):
+        """
+        You should be able to iterate over the paginator
+        object using a for loop.
+        """
+        paginator = Paginator([1, 2, 3], 2)
+        # Create a list using a for loop - this should
+        # implicitly check whether iteration works.
+        page_list = [page for page in paginator]
+        # Assert that the pages look correct
+        self.assertIn(1, page_list[0])
+        self.assertIn(3, page_list[1])
```

Comment:
In the test, can you expictly create the iterator, with `iter()`. Then `assertEqual(next(...), ...)` to the actual expected lists (`[1, 2]` and `[3]`, right?)

Reply:
Yep, makes sense.